### PR TITLE
Use multiplayer configurator to copy lobby configuration when opening an example

### PR DESCRIPTION
**Pull request by AlexandreSi**

### CODE REVIEW

File: newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as React from 'react';
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
+import { duplicateLobbyConfiguration } from '../Utils/GDevelopServices/Play';
+
+const gd = global.gd;
```
Comment (4ian) [inline newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js:6]: missing typing `: libGDevelopJs`

File: newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+type UseMultiplayerLobbyConfiguratorOutput = {
```
Comment (4ian) [inline newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js:8]: {| |}

File: newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ * opening a project from an example.
+ */
+export const useMultiplayerLobbyConfigurator = (): UseMultiplayerLobbyConfiguratorOutput => {
+  const { profile, getAuthorizationHeader } = React.useContext(
+    AuthenticatedUserContext
+  );
+  const configureMultiplayerLobbiesIfNeeded = React.useCallback(
+    async (project: gdProject, sourceGameId: string) => {
+      if (!profile) {
+        console.warn(
```
Comment (ClementPasteau) [inline newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js:26]: I'm not sure we need a warn here, if a user creates a project from an example without having created an account it shouldn't be a warn, just an early return?

**Review by 4ian — APPROVED** [review]

**Review by ClementPasteau — APPROVED** [review]

File: newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js (hunk 3 above)
Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js:26]: Oh yes sorry I'll do it differently

**Pull request merged**